src/memray/reporters/flamegraph.py

```diff
@@ -178,6 +195,11 @@ def _from_any_snapshot(
                     for interval in intervals
                 )
 
+        if inverted:
+            frames[0][
```

Comment:
Is the assignment necessary? `update_import_system_frames_for_inverted_flamegraph` does this:

```
nodes[node_index]["import_system"] |= result
```

and on the first call `node_index` is 0 so it should update frames[0] already, no?